### ISSUE TITLE
Redirect URI Should not be required

### ISSUE THREAD

**Issue by ryshep111** [okta/okta-oidc-js#425]
## I'm submitting this issue for the package(s):

- [ ] jwt-verifier
- [x] okta-angular
- [ ] oidc-middleware
- [ ] okta-react
- [ ] okta-react-native
- [ ] okta-vue

## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior
Configuration validation fails if you do not provide a redirect uri

## Expected behavior
The redirect uri will default to the current origin if not provided, thus it should not be required

## Minimal reproduction of the problem with instructions
omit the redirectUri from the config

## Environment

- Package Version:
- Browser:
- OS:
- Node version (`node -v`):
- Other:

**Comment by swiftone** [comment]
> The redirect uri will default to the current origin 

This is not true if you mean in the OIDC redirect - without an explicit (and whitelisted) uri you will remain on the issuer site and not return to the application callback uri.  (you may be thinking of our okta-auth-js library).

We are implementing revisions in our SDKs that simplify these configs, though the angular SDK has not yet been updated.  Is having a more straightforward config the purpose of your request, or is there another concern?

**Comment by ryshep111** [comment]
Yes, I was referring to the okta-auth-js library. My apologies as I did not realize there were other use cases. While we are on the subject of okta-auth-js, I did find this bug which I logged with them but I think it is actually this project: https://github.com/okta/okta-auth-js/issues/204. Thinking this can be closed but we may need a new issue to address said bug.
